src/components/board-blotter/board-blotter.module.css

```
@@ -34,6 +34,22 @@
+  appearance: none;
+  background: transparent;
+  border: 0;
+  display: inline;
+  font: inherit;
+  line-height: inherit;
+  margin: 0;
+  padding: 0;
+}
+
+.hideButton:focus-visible {
+  outline: 1px dotted currentColor;
```

Stylelint (17.12.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)